From 931084d8de214c311ac1ffafddd2b15253136170 Mon Sep 17 00:00:00 2001
From: Nadwey <nadweyyy@gmail.com>
Date: Thu, 31 Oct 2024 16:38:41 +0100
Subject: [PATCH] Add missing NeoForge to WriteManifestFromPack

---
 curseforge/packinterop/translation.go | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/curseforge/packinterop/translation.go b/curseforge/packinterop/translation.go
index 50b87df..de4fcd8 100644
--- a/curseforge/packinterop/translation.go
+++ b/curseforge/packinterop/translation.go
@@ -4,9 +4,10 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/packwiz/packwiz/core"
 	"io"
 	"os"
+
+	"github.com/packwiz/packwiz/core"
 )
 
 func ReadMetadata(s ImportPackSource) ImportPackMetadata {
@@ -93,6 +94,11 @@ func WriteManifestFromPack(pack core.Pack, fileRefs []AddonFileReference, projec
 			ID:      "forge-" + forgeVersion,
 			Primary: true,
 		})
+	} else if neoforgeVersion, ok := pack.Versions["neoforge"]; ok {
+		modLoaders = append(modLoaders, modLoaderDef{
+			ID:      "neoforge-" + neoforgeVersion,
+			Primary: true,
+		})
 	} else if quiltVersion, ok := pack.Versions["quilt"]; ok {
 		modLoaders = append(modLoaders, modLoaderDef{
 			ID:      "quilt-" + quiltVersion,

