diff --git a/curseforge/request.go b/curseforge/request.go
index d219751..1f7d74b 100644
--- a/curseforge/request.go
+++ b/curseforge/request.go
@@ -16,7 +16,7 @@ import (
 const cfApiServer = "api.curseforge.com"
 
 // If you fork/derive from packwiz, I request that you obtain your own API key.
-const cfApiKeyDefault = "JDJhJDEwJHNBWVhqblU1N0EzSmpzcmJYM3JVdk92UWk2NHBLS3BnQ2VpbGc1TUM1UGNKL0RYTmlGWWxh"
+const cfApiKeyDefault = "JDJhJDEwJGJMNGJJTDVwVVdxZmNPN0tRdG5NUmVha3d0ZkhiTktoNnYxdVRwS2x6aHdvdWVFSlFuUG5tCg"
 
 // Exists so you can provide it as a build parameter: -ldflags="-X 'github.com/packwiz/packwiz/curseforge.cfApiKey=key'"
 var cfApiKey = ""
